backend/mora/service/configuration_options.py

```diff
@@ -0,0 +1,191 @@
+#
+# Copyright (c) Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import psycopg2
+import flask
+import logging
+from mora import exceptions
+from .. import settings
+from .. import util
+
+logger = logging.getLogger("mo_configuration")
+
+blueprint = flask.Blueprint('configuration', __name__, static_url_path='',
+                            url_prefix='/service')
+
+
+if not (settings.USER_SETTINGS_DB_USER and settings.USER_SETTINGS_DB_NAME and
+        settings.USER_SETTINGS_DB_HOST and settings.USER_SETTINGS_DB_PORT and
+        settings.USER_SETTINGS_DB_PASSWORD):
+    error_msg = 'Configuration error of user settings connection information'
```

Comment:
The log output is not especially helpful. We should print the values of the variables, so we have an idea of what went wrong.